xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml

```diff
@@ -61,7 +61,9 @@
       #set ($toolBar = "#template('simpleedittoolbar.vm')")
       $!toolBar.replace('{{', '&amp;#123;&amp;#123;')
       ## Display a simple textarea.
-      &lt;textarea id="$id" cols="80" rows="25" name="$name"&gt;$escapetool.xml($editedDocument.content)&lt;/textarea&gt;
+      &lt;textarea id="$id" cols="80" rows="25" name="$name" ##
+      aria-label="$escapetool.xml($services.localization.render('platform.appwithinminutes.content.edit.label'))"&gt;##
```

Comment:
Is this really necessary? Shouldn't this always be used in a form that displays a label? What seems to be rather happening I think is that this textarea has the wrong id/the code for generating the label assumes the wrong id.

Reply:
Addressed in the latest changes by https://github.com/xwiki/xwiki-platform/pull/3584#issuecomment-2486160417 :+1: 


Note that the there was a comment to make sure the ID wouldn't be changed because it broke style/functionalities. I tried it out and everything seemed fine. With a quick search I could find that there's no specific style anymore.